BaseApiBundle/Controller/BaseController.php

```diff
@@ -64,6 +69,11 @@ protected function reverseTransform(Request $request, $id, $type, $event, $event
             $em->persist($mixed);
             $em->flush();
 
+            if(in_array('OpenOrchestra\ModelInterface\Event\EventTrait\EventStatusable', class_uses($eventClass))) {
```

Comment:
I'm not sure this classname should be hardcoded, it does not allow an easy extend of the code :/